src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationMiddleware.cs

```diff
@@ -26,6 +30,13 @@ public class CookieAuthenticationMiddleware : AuthenticationMiddleware<CookieAut
                 throw new ArgumentNullException(nameof(dataProtectionProvider));
             }
 
+            if (antiforgery == null)
```

Comment:
Why? This is a service, either it's available or your have a configuration error.